src/components/ListDetail/ListDetail.module.css

```diff
@@ -41,15 +41,28 @@
  * 1) Take up the rest of the space
  */
 .list-detail__body {
-  padding: var(--eds-size-2);
-
+  display: none;
```

Comment:
I need to add CSS comments but wanted to make sure this was an agreeable way to handle this first. the challenge is: the body of each list detail item needs to be displayed in a different container depending on breakpoint. Lg and up: div that is always visible and in the same flex column as the list. < lg: in a drawer that is hidden and only slides in when a list item is selected.